integration-test/transfer.integration-test.ts

```diff
@@ -129,6 +158,20 @@ describe("GET /transfer positive", () => {
             }
         )
     })
+    it("should return transferred placements", async () => {
+        await setupTransformations(["person", "departments", "placements"])
+        await setupTransfers(["person", "daycare", "departments"])
```

Comment:
Integration tests should break with the addition of unit manager as a prerequisite of daycares (fk constraint). Will require syncing.

Reply:
Yep, I'll fix this after unit manager merge.